hermes/views.py

```diff
@@ -78,19 +87,57 @@ def validate_brightness_unit(self, data):
             if row['brightness_unit'] not in brightness_units:
                 raise ValidationError(f'Unrecognized brightness unit. Accepted brightness units are {brightness_units}')
 
-
-class CandidateMessageSubSchema(Schema):
-    authors = fields.String()
-    main_data = fields.Nested(CandidateDataSchema)
-
-
-class CandidateMessageSchema(Schema):
+class MessageSchema(Schema):
     title = fields.String(required=True)
     topic = fields.String(required=True)
     event_id = fields.String()
     message_text = fields.String(required=True)
-    author = fields.String(required=True)
-    data = fields.Nested(CandidateMessageSubSchema)
+    submitter = fields.String(required=True)
+    authors = fields.String()
+    @property
+    @abstractmethod
+    def data(self):
+        pass
+
+
+class CandidateMessageSchema(MessageSchema):
+    data = fields.Nested(CandidateDataSchema)
+
+
+class PhotometryDataSchema(Schema):
+    target_name = fields.String()
+    ra = fields.String(required=True)
+    dec = fields.String(required=True)
+    date_observed = fields.String(required=True)
+    date_format = fields.String()
+    telescope = fields.String(required=True)
```

Comment:
What goes here? Is there an official registry of telescope codes we'd like to use like the MPC? Is this whatever name a use decides to give their telescope? Is this a diameter?

Reply:
Agreed.  Sadly there is no official general registry of telescopes (aside from the MPC).  But users can decide for themselves what to call their facilities.